docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -2435,9 +2435,18 @@ alertmanager_client:
 
 # (experimental) Enable UTF-8 strict mode. Allows UTF-8 characters in the
 # matchers for routes and inhibition rules, in silences, and in the labels for
-# alerts. It is recommended to check both alertmanager_matchers_disagree_total
-# and alertmanager_matchers_incompatible_total metrics before using this mode as
-# otherwise some tenant configurations might fail to load.
+# alerts. It is recommended that all tenants run the `migrate-utf8` command in
+# mimirtool before enabling this mode. Otherwise, some tenant configurations
+# might fail to load. To identify tenants with incompatible configurations,
+# search Mimir server logs for lines containing `Alertmanager is moving to a new
+# parser for labels and matchers, and this input is incompatible.`. To find
+# tenant configurations that are valid but contain ambiguous matchers search for
+# log lines containing `Matchers input has disagreement`. Each log line includes
+# the invalid input, a suggestion on how to fix the input (excluding ambiguous
+# matchers as these require manual correction), and the ID of the affected
+# tenant. Mimir must be run with debug level logging enabled otherwise these
+# lines will not be logged. For more information, refer to
```

Comment:
```suggestion
# tenant. You must run Mimir with debug-level logging enabled. Otherwise, these
# lines aren't logged. For more information, refer to
```